docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -578,6 +608,33 @@ Number of assemblies stored in the manifest metadata is equal to the number of M
 MVID records are used at runtime to verify that the assemblies loaded match those referenced by the
 manifest metadata representing the versioning bubble.
 
+## ReadyToRunSectionType.CrossModuleInlineInfo (v6.2+)
+The inlining information section captures what methods got inlined into other methods. It consists of a single _Native Format Hashtable_ (described below).
+
+The entries in the hashtable are lists of inliners for each inlinee. One entry in the hashtable corresponds to one inlinee. The hashtable is hashed the version resilient hashcode of the uninstantiated methoddef inlinee.
```

Comment:
Wording nit - should it be "The hashtable is hashed *with* the version resilient hashcode"? I'm not sure about the preposition but a preposition seems appropriate to me in this position.

Reply:
Fixed